config.m4

```diff
@@ -496,6 +533,7 @@ dnl }}}
 
   BSON_EXTRA_ALIGN=0
   AC_SUBST(BSON_EXTRA_ALIGN)
+  AC_SUBST(BSON_HAVE_DECIMAL128, 0)
```

Comment:
Just FYI: That flag isn't about BSON Decimal128 support though.

BSON Decimal128 is always available in libbson 1.4.0.
You'll be able to get a bson_decimal128_t from a string by calling
```
bson_decimal128_t decimal128;
if (bson_decimal128_from_string("0.1234", &decimal128)) {
    /* Conversion successful */
} else {
    /* Thats not a valid Decimal128 representation */
}
```

And you can convert that decimal128 to a string with
```
char string[BSON_DECIMAL128_STRING];
bson_decimal128_to_string (&decimal128, &string)
```

This is always available in libbson 1.4

`BSON_HAVE_DECIMAL128` is a check for `_Decimal128` native type.

This allows you to get a native `_Decimal128` type from `bson_decimal128_t` to use with math libraries that support that. And, to convert a `_Decimal128` into `bson_decimal128_t` for storage in MongoDB.

Similarly, you'll want on your `BSON\Decimal128` class to provide, when available:

```
$decimal128->getAsGmp();
$decimal128->getAsBCMath();
```